.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,12 +12,11 @@
 
 ### Checklists
 
-<!-- Note on checklists:
+<!-- Note:
      If you think some of these steps are not necessary for your PR,
-     just remove those checkboxes, or mark them as checked. Otherwise,
-     if some checkboxes are left unmarked, we might assume that your PR
-     is not ready to be merged and we might keep you waiting -->
-
+     remove those checkboxes, or mark them as checked. If you keep unchecked checkboxes,
+     we will assume that your PR is not ready to be merged  -->
```

Comment:
just so the people pay a bit more attention to this, it won't change anything in our review process